src/pyrad_proc/pyrad/io/mxpol_config.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hi Floor,
In rad4alp we do not have a noise subtraction on the signal for RhoHV and therefore we call it uncorrected_cross_correlation_ratio. When we correct for noise we call it cross_correlation_ratio. We reserve the corrected_ keyword for more high level corrections such as clutter suppression, attenuation correction, etc. The terminology is a bit slopy I am afraid.